src/SWP/Bundle/MultiTenancyBundle/EventListener/TenantSubscriber.php

```diff
@@ -75,8 +76,10 @@ protected function addTenant(LifecycleEventArgs $args)
                 return;
             }
 
-            if (null === $this->tenantContext) {
-                $this->tenantContext = $this->container->get('swp_multi_tenancy.tenant_context');
+            if ($this->phpcrEnabled) {
```

Comment:
```When PHPCR ODM persistence backend is enabled it will rely on tenant's unique code instead of the tenant id.
In this case, if the tenant exists in the context and the tenant code is 123abc, it will add ``WHERE tenant_id = 123abc`` to every select query.```

If you are using ORM for example, you would want to make a relation to tenant. If you use phpcr and you want to make some entities tenant aware you cant make a relations between ORM and PHPCR that's why a code is used.